### ISSUE TITLE
[Bug]: Backups is not respecting "Numbers of backups to keep" number

### ISSUE THREAD

**Issue by TeroKeso** [advplyr/audiobookshelf#1041]
### Describe the issue

Backups is not respecting "Numbers of backups to keep" number. Maximum backups to keep (in GB) is working and removing older backups when making new versions.   

![image](https://user-images.githubusercontent.com/29271631/193800095-3aa90c7c-704a-4d75-9b70-6f77155df199.png)

INFO
[BackupManager] 45 Backups Found
DEBUG
[BackupManager] Removing old backup 2022-08-19T0100
2022-10-03 13:01:59
DEBUG
[BackupManager] Removing Backup "/metadata/backups/2022-08-19T0100.audiobookshelf"
2022-10-03 13:01:59
INFO
[BackupManager] Backup "2022-08-19T0100" Removed 
"2022-10-03 13:03:11
DEBUG
[BackupManager] Removing old backup 2022-08-20T0100
2022-10-03 13:03:11
DEBUG
[BackupManager] Removing Backup "/metadata/backups/2022-08-20T0100.audiobookshelf""

### Steps to reproduce the issue

1. Activate Backups
2. Keep defaults 
3. Create Backups button. 

### Audiobookshelf version

2.2.0

### How are you running audiobookshelf?

Docker

**Comment by advplyr** [comment]
This is only removing 1 backup for every 1 backup that is added if the number of backups exceeds the max you set.
So if you set the max number of backups to 5 when you already had 10 backups then you will always have 10 backups. I don't remember exactly why I implemented it this way but I suppose it could be updated to loop through and remove all the ones exceeding the max.
Am I right in suggesting that you set the max to 5 after you already had more then 5?

**Comment by TeroKeso** [comment]
i have been running software from early days. Did not change the defaults because I am backupping server regularly. I just activated log time past versions it and have it be.  I did notice slow size growth backups did find this. After removing extra backups it is working with the default values.

In long run I think this is a good compromise for data preservation. Maybe a notification in the page that it will not remove old backups if changed.
